flybywiresim/a32nx
# fix(cond): pack pushbutton fault light

## Summary of Changes

The introduction of #6220 removed the pack pushbutton FAULT light illumination when the position of the valve disagreed with the commanded position. This PR re-introduces that functionality (to the best of my knowledge).

Now the FAULT light will illuminate when the position of the pack flow valve disagrees with the commanded position. This is mainly seen on the ground when there is no bleed air available and the pack pushbutton is pushed in (ON position, white light extinguished). In the future when failures are included for this system there will be other scenarios where the real position and the commanded positions of the valve will disagree.

## Screenshots (if necessary)



![image](https://user-images.githubusercontent.com/60938089/167506959-d2a0d09a-0ce4-499c-bb71-52bb0c42398c.png)

## References


See issue #1088

See conversation on Discord #ata-21-pressurisation

Also FCOM extract for reference:

![image](https://user-images.githubusercontent.com/60938089/167505057-cbd52e33-d1fd-4cc6-bb9b-6af4ccec961b.png)




## Additional context

No ECAM warnings have been included in this PR. For future work when the new FWC is implemented.


Discord username (if different from GitHub): Commander Ga#1548

## Testing instructions


1. Start cold and dark and power up the aircraft.
2. Ensure the pack pushbuttons are in the ON position (no white lights).
3. Check that the FAULT light illuminates with engines off and APU off.
4. When APU bleed is switched on, the FAULT light extinguishes.
5. Even after APU or engines are on, removing the bleed from the packs (ENG 1/2 BLEED or APU BLEED pb are off) illuminates the FAULT light.
6. Ensure the COND and BLEED systems behave normally.

Please note: on the ground without bleed the valve remains closed but commanded position is open (that's why the FAULT light illuminates). If you switch the engine start mode to IGN, or press the DITCHING pushbutton on the overhead, the commanded position will be closed (as per the FCOM above), and the FAULT light will extinguish, even without bleed.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
